doc/USING.rst

```diff
@@ -168,12 +179,12 @@ Note: the following command assumes you `downloaded the pre-trained model <#gett
 
    ./deepspeech --model deepspeech-0.7.0-models.pbmm --scorer deepspeech-0.7.0-models.scorer --audio audio_input.wav
 
-See the help output with ``./deepspeech -h`` and the :github:`native client README <native_client/README.rst>` for more details.
+See the help output with ``./deepspeech -h`` for more details.
 
 Installing bindings from source
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If pre-built binaries aren't available for your system, you'll need to install them from scratch. Follow these :github:`native client installation instructions <native_client/README.rst>`.
+If pre-built binaries aren't available for your system, you'll need to install them from scratch. Follow the :github:`native client build and installation instructions <native_client/README.rst>`.
```

Comment:
Do we want to/can we not link to GitHub here?

Reply:
I think ideally we move the build instructions to ReadTheDocs as well.